sspl_test/run_qa_test.sh

```diff
@@ -77,13 +78,13 @@ pre_requisites()
         if [ -f /var/$PRODUCT_FAMILY/sspl/data/iem/last_processed_msg_time ]; then
             $sudo mv /var/$PRODUCT_FAMILY/sspl/data/iem/last_processed_msg_time /var/$PRODUCT_FAMILY/sspl/orig-data/iem/last_processed_msg_time
         fi
-    fi
 
-    # Enable ipmi simulator
-    if [ "$IS_VIRTUAL" == "true" ]
-    then
+        # Enable ipmi simulator
         cp -Rp $script_dir/ipmi_simulator/ipmisimtool /usr/bin
-        touch /tmp/activate_ipmisimtool
+        mkdir -p "$DATA_PATH/server/"
+        chown -R sspl-ll:sspl-ll "$DATA_PATH/server/"
+        chmod 755 "$DATA_PATH/server/"
```

Comment:
Use `chmod` with symbolic notation if this make sense to you: [Chmod by Letters Vs Numbers](https://unix.stackexchange.com/questions/94212/chmod-by-letters-vs-numbers)

Reply:
Hi @jjxsg , I tried to think on this, but now I am confused what permissions to give and how the cmd should be, can you suggest something?

Reply:
`chmod u=rwx, g=rw, o=rw  "$DATA_PATH/server/"`  ??

Reply:
OR `chmod a=rw, u+x "$DATA_PATH/server/"`  this may be.

Reply:
It should be execute for group and others 
`7=rwx 5=r-x 5=r-x`

Reply:
`chmod 755 "$DATA_PATH/server/"` looks better to me than `chmod a=rx,u+w "$DATA_PATH/server`.
if any thinks other one is better I will change it, so please put your thoughts.